src/en/news/blog/2008/delayed-capability-release/index.md

```diff
@@ -0,0 +1,42 @@
+---
+title: "Delayed capability release"
+date: "2008-04-18"
+author: "sage"
```

Comment:
Would it be possible to update to the full name and capitalise the author for all the articles?

Reply:
We can clean these up in a follow-on PR?

Reply:
Yep, happy to go with usernames for launch. We'll leave to a future PR to revisit 👍